sdk/ml/azure-ai-ml/azure/ai/ml/_restclient/v2023_04_01_preview/operations/_featureset_versions_operations.py

```diff
@@ -1060,7 +1060,10 @@ def prepare_request(next_link=None):
                 )
                 request = _convert_request(request)
                 request.url = self._client.format_url(request.url)
-                request.method = "GET"
+                # ---------------------- PATCH ----------------------
```

Comment:
This is usually not a good idea. I can sign off if
1. We also have e2e tests to prove this is working as expected.
2. We actively work on the proper solution and remove this hack as soon as the proper solution is ready.

Reply:
1. We have a PR for e2e test. I will add one more test for materialization job list there. https://github.com/Azure/azure-sdk-for-python/pull/29423/files
2. We have ongoing PR for autorest update, after which we can patch it through patch file. https://github.com/Azure/azure-sdk-for-python/pull/29930